target-gen/src/commands/test.rs

```diff
@@ -43,8 +43,13 @@ pub fn cmd_test(
 
     let file = File::open(Path::new(definition_export_path))?;
     probe_rs::config::add_target_from_yaml(file)?;
+    let file_handle = File::open(Path::new(definition_export_path))?;
```

Comment:
We already open this file two lines above :) We can reuse that handle I think :)

Reply:
Yeah, I am a bit of a rust newbie, so unable to figure this one out gracefully.

`probe_rs::config::add_target_from_yaml(file)?;` moves file into `add_target_from_yaml`, but I also want to call `serde_yaml::from_reader(file)` which also moves ownership.

Reply:
Ah fair, I guess we can open it twice :) I did not realize its obviously moved :) I mean reading the file will also move the cursor so we can also just reopen instead of moving it back.

Reply:
 Sounds good. I believe I have a workable solution now, let me know what you think.